drivers/sensors/cirque_pinnacle.c

```diff
@@ -213,7 +213,12 @@ void cirque_pinnacle_init(void) {
     // Host sets z-idle packet count to 5 (default is 30)
     RAP_Write(Z_IDLE_COUNT, Z_IDLE_COUNT_VALUE);
 
-    cirque_pinnacle_set_adc_attenuation(0xFF);
+#ifdef CIRQUE_PINNACLE_ATTENUATION
+    cirque_pinnacle_set_adc_attenuation(CIRQUE_PINNACLE_ATTENUATION);
+#else
+    cirque_pinnacle_set_adc_attenuation(ADC_ATTENUATE_4X);
+#endif
+
```

Comment:
Please move this into a single define in the headers file, with the given default value, similar to this https://github.com/qmk/qmk_firmware/blob/master/drivers/sensors/cirque_pinnacle.h#L23-L25

Reply:
@Kriechi If I do that, where would the user be able to set their values for it?

Reply:
Actually, if anything: 
```suggestion
    cirque_pinnacle_set_adc_attenuation(CIRQUE_PINNACLE_ATTENUATION);
```

And add this above:
```c
#ifndef CIRQUE_PINNACLE_ATTENUATION
#    define CIRQUE_PINNACLE_ATTENUATION ADC_ATTENUATE_4X
#endif
```